src/compiler/transformers/decorators-to-static/event-decorator.ts

```diff
@@ -27,12 +27,22 @@ export const eventDecoratorsToStatic = (
   }
 };
 
+/**
+ * Parse a single instance of Stencil's `@Event()` decorator and generate metadata for the class member that is
+ * decorated
+ * @param diagnostics a list of diagnostics used as a part of the parsing process. Any parse errors/warnings shall be
+ * added to this collection
+ * @param typeChecker an instance of the TypeScript type checker, used to generate information about the `@Event()` and
+ * its surrounding context in the AST
+ * @param prop the property on the Stencil component class that is decorated with `@Event()`
+ * @returns generated metadata for the class member decorated by `@Event()`
```

Comment:
should we indicate the possible `null` return type here in `@returns` ?

Reply:
👍 done in 5c296cfb90aca387ab6ccf9fbf043be3fd444f73